ibm_catalog.json

```
@@ -443,7 +444,7 @@
               "key": "enable_platform_metrics",
               "type": "string",
               "default_value": "true",
```

@Ak-sky - I think we can remove the workaround now, as per latest changes if not done already
```suggestion
              "type": "boolean",
              "default_value": true,
```
And hence the optional values should also be updated from string to bool i.e. `"true" -> true and "false" -> false`